grafana_client/elements/dashboard.py

```diff
@@ -15,6 +15,16 @@ def get_dashboard(self, dashboard_uid):
         get_dashboard_path = "/dashboards/uid/%s" % dashboard_uid
         r = self.client.GET(get_dashboard_path)
         return r
+    
+    def get_dashboard_by_name(self, dashboard_name):
+        """
+
+        :param dashboard_name:
+        :return:
+        """
+        get_dashboard_path = "/dashboards/db/%s" % dashboard_name
+        r = self.api.GET(get_dashboard_path)
```

Comment:
Hi Karsten,

the test case you added croaks on this line with `AttributeError: 'Dashboard' object has no attribute 'api'`, see [^1]. While refactoring the package, we slightly changed its internals, so on this spot, `self.api` became `self.client`:

```suggestion
        r = self.client.GET(get_dashboard_path)
```

Can you amend your patch correspondingly? If anything goes right after that, the tests should succeed. In order to verify it, you can run the test suite on your workstation, like documented in the README at [^2].

With kind regards,
Andreas.

[^1]: https://github.com/panodata/grafana-client/runs/5128322276
[^2]: https://github.com/panodata/grafana-client#software-tests